### DISCUSSION TITLE
Audacity can't record mono when using Equalizer APO

### DISCUSSION THREAD

**Discussion by futag** [audacity/audacity#4899] [Q&A]
### Steps to reproduce

1. Install [Equalizer APO](https://sourceforge.net/projects/equalizerapo/).
2. Install your microphone as an APO in Configurator.exe.  
![image](https://user-images.githubusercontent.com/21020026/236585191-b6284aa7-d895-4cbd-b3f7-482e7988716f.png)
3. Set a random VST plugin.
4. Open Audacity.
5. Set the recording device as the microphone with the APO installed.  
![image](https://user-images.githubusercontent.com/21020026/236585334-99255d91-c4eb-4819-b049-a490a15c17ba.png)
6. Set channel to mono

### Expected behavior

When I first click on record in Audacity, it shows an error (-9999) and doesn't record. For some reason, after clicking on record multiple times, it eventually starts recording, but the microphone is not filtered in the APO editor. To resolve this, I have to open Configurator.exe, click OK, and restart. However, Audacity still cannot record and the issue keeps repeating.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by dozzzzer** [comment]
I couldn't reproduce the issue on Windows 11 using the latest version 3.2.2.

**Comment by futag** [comment]
Ok, maybe error when convert channel to mono can make error like this

**Comment by dozzzzer** [comment]
I've tried this time on Windows 10 laptop and Audacity 3.3.2 and could reproduce the issue with none of the two input devices selected, neither with 1 nor with 2 channels selected in Audacity.

**Comment by scclie** [comment]
I had the same problem using an external audio card with two inputs, which were in the form of one stereo device.  Audacity, for some reason, can't convert such two tracks to mono - to solve it, all you need to do is to set the recording to stereo in the settings and manually cut off the unwanted track. But if you don't use the equalizer apo, you don't have this problem.
